generators/server/templates/_pom.xml

```diff
@@ -1221,7 +1206,7 @@
                         <configuration>
                             <failOnMissingWebXml>false</failOnMissingWebXml>
                             <%_ if (!skipClient) { _%>
-                            <warSourceDirectory><% if (clientFramework !== 'angular1') { %><%= CLIENT_DIST_DIR %><% } else { %><%= CLIENT_MAIN_SRC_DIR %><% } %></warSourceDirectory>
+                            <warSourceDirectory><%= CLIENT_MAIN_SRC_DIR %></warSourceDirectory>
```

Comment:
@deepu105 : I think the correct value should be `CLIENT_DIST_DIR`, according to the condition `!== 'angular1'`